examples/prism-dockerfile.html

```diff
@@ -0,0 +1,39 @@
+<h1>Dockerfile</h1>
+<p>To use this language, use the class "language-docker".</p>
+
+<h2>Comments</h2>
+<pre><code># These are the comments for a dockerfile.
+# I want to make sure $(variables) don't break out,
+# and we shouldn't see keywords like ADD or ENTRYPOINT
+</code></pre>
+
+<h2>Instructions</h2>
+<pre><code># Use the official go docker image built on debian.
+FROM golang:1.4.2
+
+# Grab the source code and add it to the workspace.
+ADD . /go/src/github.com/JustinBeckwith/revel-appengine
+
+# Install revel and the revel CLI.
+RUN go get github.com/revel/revel
+RUN go get github.com/revel/cmd/revel
+</code></pre>
+
+
+<h2>Full example</h2>
```

Comment:
The "Instructions" and the "Full example" look more or less the same for me. I guess you could remove the "Instructions" one.
Also, I find the official example more complete than yours. https://docs.docker.com/reference/builder/#dockerfile-examples